dev/com.ibm.ws.transport.http/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -123,6 +127,10 @@
 			ibm:final="true" required="false" type = "String"
 			default="(|(service.pid=${samesiteRef})(&amp;(service.factoryPid=com.ibm.ws.http.samesite)(|(id=com.ibm.ws.http-${id}/*)(id=defaultSameSite))))" />
     
+    	<AD id="headers.target" name="internal" description="internal use only"
+			ibm:final="true" required="false" type = "String" ibm:beta="true"
+			default="(|(service.pid=${headersRef})(&amp;(service.factoryPid=com.ibm.ws.http.headers)(|(id=com.ibm.ws.http-${id}/*)(id=defaultHeaders))))" />
```

Comment:
Formatting seems a bit odd here as well.

Reply:
For this one, all the other elements have the same format, so leaving this one as is for now.

Reply:
See the other metatype.xml comment for proposed solution.